docs/source/micromamba.md

```diff
@@ -84,12 +84,12 @@ micromamba activate
 micromamba install python=3.6 jupyter -c conda-forge
 ```
 
-## Windows
+#### Windows
 
 Micromamba also has Windows support! For Windows, we recommend powershell. Below are the commands to get micromamba installed in `PowerShell`.
 
-```pwsh
-Invoke-Webrequest -URI https://micromamba.snakepit.net/api/micromamba/win-64/latest -OutFile micromamba.tar.bz2
+```powershell
+Invoke-Webrequest -URI https://micro.mamba.pm/api/micromamba/win-64/latest -OutFile micromamba.tar.bz2
```

Comment:
Nitpick comment, feel free to ignore: did you consider to host directly the static linked binaries as `https://micro.mamba.pm/api/micromamba/win-64/latest/micromamba.exe` or something similar? 
This would simplify the oneliner to download micromamba to: 
~~~
curl -O https://micro.mamba.pm/api/micromamba/win-64/latest/micromamba.exe
~~~
(curl is installed by default since 2017 on Windows)

Reply:
curl in powershell is actually not a real curl :)

yes, definitely I am considering it. Currently the `micro.mamba.pm` endpoint forwards API requests to Anaconda.org, and you download the exact build output from the conda-forge builds. Happy to reconsider this pipeline sooner or later.

Reply:
curl in powershell, under the hood, is Invoke-Webrequest afaik.

Reply:
> curl in powershell is actually not a real curl :)

FAQ, you are right. I had no idea. Basically "curl" curl is shipped in Windows, i.e. in a command prompt you can: 
~~~
C:\Users\STraversaro>where curl
C:\Windows\System32\curl.exe

C:\Users\STraversaro>curl --version
curl 7.55.1 (Windows) libcurl/7.55.1 WinSSL
Release-Date: 2017-11-14, security patched: 2019-11-05
Protocols: dict file ftp ftps http https imap imaps pop3 pop3s smtp smtps telnet tftp
Features: AsynchDNS IPv6 Largefile SSPI Kerberos SPNEGO NTLM SSL
~~~ 

But instead in Powershell the real curl is shadowed by the "fake" curl:
~~~
(base) PS C:\Users\STraversaro> where curl
(base) PS C:\Users\STraversaro> curl --version
curl : The remote name could not be resolved: '--version'
At line:1 char:1
+ curl --version
+ ~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand

(base) PS C:\Users\STraversaro>
~~~

-_-

Reply:
Apparently the issue was solved in Powershell 6, but there is no plan to switch the default Powershell in Windows from Powershell 5 to Powershell 6. (-_-)^2